apphelpers/rest/hug.py

```diff
@@ -288,15 +327,23 @@ def multisite_access_wrapper(f):
                 def wrapper(request, *args, **kw):
 
                     user = request.context["user"]
+                    site_id = (
+                        int(kw[self.site_identifier])
+                        if self.site_identifier in kw
+                        else None
+                    )
 
                     # this is authentication part
                     if not user.id:
                         raise HTTPUnauthorized("Invalid or expired session")
 
+                    # bound site authorization
+                    if user.site_ctx and site_id != user.site_ctx:
```

Comment:
@gauravr @shon  Due to this condition for APIs from custom domains, we are getting the error "Invalid or expired session". 
I think, we are expecting site_id from kw (#330) and we dont have site_id in many APIs e.g. users/{id}/subscriptions   and login_required is true for this url's callback function.
One possible solution would be to exclude this check, but need to check implications of that? 
Can we pls discuss this whenever you have time? This is bit urgent

Reply:
I mean this check `site_id != user.site_ctx`

Reply:
@gauravr I am not facing this issue anymore Today. Yesterday, We were facing the issue just after the custom domain linking and will observe more Today. I think this is not the release blocker so once you back we will investigate more if needed cc: @shon